docusaurus/docs/dev-docs/api/rest/relations.md

```diff
@@ -12,32 +12,48 @@ tags:
 
 import NotV5 from '/docs/snippets/_not-updated-to-v5.md'
 
-# Managing relations through the REST API
-
-<NotV5 />
+# Managing relations with API requests
 
 Defining relations between content-types (that are designated as entities in the database layers) is connecting entities with each other.
 
-Relations between content-types can be managed through the [admin panel](/user-docs/content-manager/managing-relational-fields#managing-multiple-choices-relational-fields) or through [REST](/dev-docs/api/rest) requests sent to the Content API.
+Relations between content-types can be managed through the [admin panel](/user-docs/content-manager/managing-relational-fields#managing-multiple-choices-relational-fields) or through [REST API](/dev-docs/api/rest) or [Document Service API](/dev-docs/api/document-service) requests.
 
 Relations can be connected, disconnected or set through the Content API by passing parameters in the body of the request:
 
-|  Parameter name | Description | Type of update |
-|-----------------|------------------|----------------|
-| [`connect`](#connect)       | Connects new entities.<br /><br />Can be used in combination with `disconnect`.<br /><br />Can be used with [positional arguments](#relations-reordering) to define an order for relations.    | Partial
-| [`disconnect`](#disconnect)    | Disconnects entities.<br /><br />Can be used in combination with `connect`. | Partial
-| [`set`](#set)           | Set entities to a specific set. Using `set` will overwrite all existing connections to other entities.<br /><br />Cannot be used in combination with `connect` or `disconnect`.  | Full
+|  Parameter name         | Description | Type of update |
+|-------------------------|-------------|----------------|
+| [`connect`](#connect)   | Connects new entities.<br /><br />Can be used in combination with `disconnect`.<br /><br />Can be used with [positional arguments](#relations-reordering) to define an order for relations.    | Partial |
+| [`disconnect`](#disconnect)    | Disconnects entities.<br /><br />Can be used in combination with `connect`. | Partial |
+| [`set`](#set)           | Set entities to a specific set. Using `set` will overwrite all existing connections to other entities.<br /><br />Cannot be used in combination with `connect` or `disconnect`.  | Full |
+
+:::note
+When [Internationalization (i18n)](/user-docs/content-manager/translating-content) is enabled on the content-type, you can also pass a locale to set relations for a specific locale, as in this Document Service API example:
+
+```js
+await strapi.documents('api::restaurant.restaurant').update({ 
+  documentId: 'a1b2c3d4e5f6g7h8i9j0klm',
+  locale: 'fr',
```

Comment:
@Marc-Roig I'm not sure about this example. I did it by memory, trying to remember the exact syntax from our call. 😅

Reply:
yep! this is a valid syntax :)